CMakeLists.txt

```diff
@@ -174,7 +174,9 @@ message("  CMAKE_INSTALL_FULL_INCLUDEDIR           : ${CMAKE_INSTALL_FULL_INCLUD
 message("  CMAKE_INSTALL_FULL_LIBDIR               : ${CMAKE_INSTALL_FULL_LIBDIR}")
 message("  CMAKE_INSTALL_FULL_DATADIR              : ${CMAKE_INSTALL_FULL_DATADIR}")
 message("Enable:")
-message("  ENABLE_OPENCL                           : ${ENABLE_OPENCL}")
+if(DEFINED OPENCL_FOUND)
+  message("  ENABLE_OPENCL                           : ${ENABLE_OPENCL}")
+endif()
```

Comment:
@dvrogozh 
I hope that it is better...

I tried to "play" with something as:
```
option( ENABLE_OPENCL "Build targets dependent on OpenCL?" FALSE )
                                                                                                           ^
```
But it is all time defined as TRUE... I don\`t know why possible I did it wrong...

Reply:
Or I can implement:
```
if(DEFINED OPENCL_FOUND)
  message("  ENABLE_OPENCL                           : ${ENABLE_OPENCL}")
else()
  message("  ENABLE_OPENCL                           : OFF")
endif()
```

Output:
```
Enable:
  ENABLE_OPENCL                           : OFF
  ENABLE_X11_DRI3                         : OFF
  ENABLE_WAYLAND                          : OFF
  ENABLE_ITT                              : OFF
  ENABLE_TEXTLOG                          : OFF
  ENABLE_STAT                             : OFF
Build:
...
```

Reply:
@onabiull Your opinion on this https://github.com/Intel-Media-SDK/MediaSDK/pull/1022#discussion_r245482645? Is it ready to merge? Or should I to re-implement as in https://github.com/Intel-Media-SDK/MediaSDK/pull/1022#discussion_r245483252 ?